doc/user/container.md

```diff
@@ -117,11 +117,143 @@ versions.
 Similar to POSIX extended attributes, users can attach some metadata to each
 container through the daos_cont_{list/get/set}_attr() API.
 
-## Container ACLs
+## Container Access Control Lists
 
-Support for per-container ACLs is scheduled for DAOS v1.2. Similar to pool ACLs,
-container ACLs will implement a subset of the NFSv4 ACL standard. This feature
-will be documented here once available.
+Client user and group access for containers is controlled by 
+[Access Control Lists (ACLs)](/doc/user/acl.md).
+
+Access-controlled container accesses include:
+* Opening the container for access.
```

Comment:
Only a suggestion, but the way I organize my thoughts around accesses is in 3 categories: data, metadata, other. What if it were presented something like this?

Opening the container for access
Deleting the container
Reading and writing data (objects)
Reading and writing metadata
- Getting and setting container properties.
- Getting, setting, and listing user attributes.
- Getting, setting, and listing snapshots.
- Getting and modifying the container ACL.
- Modifying the container's owner.

Reply:
Your proposed organization definitely makes sense - my concern was in communicating how the accesses were determined. Snapshots and user attributes are gated under the umbrella of Read-data (r) and Write-data (w) permissions, while properties, ACL, and owner metadata each have their own permission bits that must be set for access. Maybe there is another way I can communicate this, though.

Reply:
I didn't modify this on this draft - still thinking if there's another good way to communicate this. Please have a look at the changes and let me know what you think.

Reply:
I see now - the order in which it is presented is roughly aligned to the acl.md table content "Container Meaning" column.This makes sense, so I would say keep it the way it is currently presented.